columnflow/calibration/jets_coffea.py

```diff
@@ -0,0 +1,646 @@
+# coding: utf-8
+
+"""
+Calibration methods for jets
+"""
+
+import os
+
+import law
+
+from columnflow.util import maybe_import, memoize
+from columnflow.calibration import Calibrator, calibrator
+from columnflow.calibration.util import propagate_met, ak_random
+from columnflow.production.util import attach_coffea_behavior
+from columnflow.columnar_util import set_ak_column
+
+np = maybe_import("numpy")
+ak = maybe_import("awkward")
+coffea = maybe_import("coffea")
+
+coffea_extractor = maybe_import("coffea.lookup_tools.extractor")
+coffea_jetmet_tools = maybe_import("coffea.jetmet_tools")
+coffea_txt_converters = maybe_import("coffea.lookup_tools.txt_converters")
```

Comment:
Is the main coffea package required here? If not, we could remove that single line, but if it is needed for the following three subpackage imports to work, we could remove the assignment, i.e.,

```python
maybe_import("coffea")
```

Reply:
Good point. It doesn't look like the main package is required explicitly, so I removed the line.